Task1.PianoKeyboard/Enums/EAccidental.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Task1.PianoKeyboard.Enums
+{
+    internal enum EAccidental
```

Comment:
Do not use "E" prefix, it is not common practice, https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces